### PR TITLE
Init mingw-w64-ossia-score package

### PR DESCRIPTION
**Pull request by jcelerier**
ossia score ([ossia.io](https://ossia.io/)) is a software for creating live audio, video, light & multimedia shows and art installations.
Note: the last current stable release needs some patches for building. Next version will work directly, I'll update the PR asap.

### CODE REVIEW

File: mingw-w64-ossia-score/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+  "${MINGW_PACKAGE_PREFIX}-qt6-tools"
+  "${MINGW_PACKAGE_PREFIX}-portaudio"
+  "${MINGW_PACKAGE_PREFIX}-fftw"
+  "${MINGW_PACKAGE_PREFIX}-ffmpeg"
+  "${MINGW_PACKAGE_PREFIX}-SDL2"
+)
+makedepends=(
+  "${MINGW_PACKAGE_PREFIX}-cmake"
+  "${MINGW_PACKAGE_PREFIX}-ninja"
+  "${MINGW_PACKAGE_PREFIX}-gcc"
```
Comment (Biswa96) [inline mingw-w64-ossia-score/PKGBUILD:30]:
```suggestion
  "${MINGW_PACKAGE_PREFIX}-cc"
```

**Comment by luzpaz** [comment]
What's left to do here ?

**Comment by jcelerier** [comment]
I have to tag a new ossia score release as the 3.1.11 did not build as is with this pkgbuild.. I have a few more fixes I want to get in and I tag it & update this

**Comment by jcelerier** [comment]
let's try with 3.1.12

**Comment by luzpaz** [comment]
soft bump

**Comment by jcelerier** [comment]
@Biswa96 any chance of review?

**Comment by luzpaz** [comment]
3.1.13 released (https://github.com/ossia/score/releases/tag/v3.1.13)

**Review by Biswa96 — APPROVED** [review]

**Comment by Biswa96** [comment]
This package fails to build in clangarm64 environment due to some issue in bundled rubberband project. The full build log can be found here https://packages.msys2.org/queue. Would it be possible to use system libraries instead of those bundled ones? There are many such bundled libraries in the ossia-score project.

**Comment by jcelerier** [comment]
yes, there's a CMake flag to toggle use of system libraries if available, I'll set it. thanks !!!

**Comment by jcelerier** [comment]
I cannot find the build log there, clangarm64 queue seems empty. @Biswa96 do you have a direct link through it?

**Comment by Biswa96** [comment]
> I cannot find the build log there, clangarm64 queue seems empty.

clangarm64 in ossia-score has been disabled recently to clear the build queue. The error is exactly here https://github.com/msys2-arm/msys2-autobuild/actions/runs/8220020248/job/22478665302#step:11:4548

**Comment by jcelerier** [comment]
@Biswa96 I updated the package to add a flag that makes it look for host system libraries first, will push with the next version

**Pull request merged**